### PR TITLE
[Android] fix bars on world download page

### PR DESCRIPTION
**Pull request by RedAuburn**
First impressions count 😉

before/after
bitmap

### CODE REVIEW

File: android/app/src/main/java/app/organicmaps/DownloadResourcesLegacyActivity.java [hunk 1]
```diff
@@ -184,6 +184,7 @@ public void onProgress(String countryId, long localSize, long remoteSize)
   protected void onSafeCreate(@Nullable Bundle savedInstanceState)
   {
     super.onSafeCreate(savedInstanceState);
+    UiUtils.setLightStatusBar(this, true);
```
Comment (biodranik) [inline android/app/src/main/java/app/organicmaps/DownloadResourcesLegacyActivity.java:187]: Should this screen be dark if the dark theme is enabled? Won't it blind up a user who downloaded OM at night for the first time?

Comment (RedAuburn) [reply android/app/src/main/java/app/organicmaps/DownloadResourcesLegacyActivity.java:187]: they'll likely just be blinded by the main map instead, but i'll make this use the system dark mode tomorrow 👍️

Comment (biodranik) [reply android/app/src/main/java/app/organicmaps/DownloadResourcesLegacyActivity.java:187]: There is a pending issue to sync with iOS version: add Use system theme setting (and make it the default for new users). And rename the current Auto to something like "Dark mode when navigating at night"

Comment (RedAuburn) [reply android/app/src/main/java/app/organicmaps/DownloadResourcesLegacyActivity.java:187]: definitely, however i think it'd be best to improve the dark mode map style before setting it by default (via using system theme). it still has a lot of work to be fully usable

**Review by biodranik — APPROVED** [review]
Is there already an issue that the first launch screen should respect default system theme setting?

**Comment by Jean-BaptisteC** [comment]
Fullscreen is beautiful 🤩

**Pull request merged**